litmus-portal/frontend/src/views/Settings/TeamingTab/Team/removeMemberModal.tsx

```diff
@@ -0,0 +1,132 @@
+import { useMutation } from '@apollo/client';
+import { IconButton, Typography } from '@material-ui/core';
+import { ButtonFilled, ButtonOutlined, Modal } from 'litmus-ui';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Loader from '../../../../components/Loader';
+import { GET_PROJECT, REMOVE_INVITATION } from '../../../../graphql';
+import { MemberInvitation } from '../../../../models/graphql/invite';
+import { Member } from '../../../../models/graphql/user';
+import { getProjectID } from '../../../../utils/getSearchParams';
+import useStyles from './styles';
+
+interface RemoveMemberModalProps {
+  row: Member;
+  showModal: () => void;
+  handleClose: () => void;
+  open: boolean;
+  isRemove: boolean;
+}
+const RemoveMemberModal: React.FC<RemoveMemberModalProps> = ({
+  row,
+  showModal,
+  handleClose,
+  open,
+  isRemove,
+}) => {
+  const classes = useStyles();
+  const projectID = getProjectID();
+
+  const { t } = useTranslation();
+
+  // mutation to remove member
+  const [removeMember, { loading }] = useMutation<MemberInvitation>(
+    REMOVE_INVITATION,
+    {
+      onCompleted: () => {
+        showModal();
+        handleClose();
+      },
+      refetchQueries: [
+        {
+          query: GET_PROJECT,
+          variables: { projectID },
+        },
+      ],
+    }
+  );
+
+  return (
+    <Modal
+      data-cy="modal"
+      open={open}
+      width="50%"
+      height="70%"
+      disableBackdropClick
+      disableEscapeKeyDown
+      onClose={() => {
+        showModal();
+        handleClose();
+      }}
+      modalActions={
+        <div className={classes.closeModal}>
+          <IconButton
+            onClick={() => {
+              showModal();
+              handleClose();
+            }}
+          >
+            <img src="./icons/closeBtn.svg" alt="close" />
+          </IconButton>
+        </div>
+      }
+    >
+      <div className={classes.body}>
+        <img src="./icons/userDel.svg" alt="lock" />
+        <div className={classes.text}>
+          <Typography className={classes.typo} align="center">
+            {isRemove
+              ? t('settings.teamingTab.deleteUser.header')
+              : t('settings.teamingTab.deleteModal.header')}
+            <strong> {row.user_name}?</strong>
```

Comment:
the classname could have had this fontWeight as bold instead

Reply:
Is there any specific reason behind not using `<strong>` ?